docs/index.rst

```diff
@@ -25,11 +25,11 @@ Standard pip install
 
    pip install stactools
 
-To install all available sub-packages, use
+Subpackages are available from PyPI:
```

Comment:
Only core subpackages, correct?

Reply:
Non-core subpackages could be on PyPI too I suppose? Maybe we should have an example of installing one that's not on PyPI to support the "third party vendor's package" case.